app/uk/gov/hmrc/cgtpropertydisposalsfrontend/models/onboarding/audit/BusinessPartnerRecordNameMatchDetails.scala

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.cgtpropertydisposalsfrontend.models.onboarding.audit
+
+import julienrf.json.derived
+import play.api.libs.json.OFormat
+
+sealed trait BusinessPartnerRecordNameMatchDetails extends Product with Serializable
```

Comment:
isn't this already in master? This is the thing whose format instance I changed this morning

Reply:
this branch was not forked from master - when the actual PR is created it will show all the conflicts.